code/modules/clothing/spacesuits/spacesuits.dm

```diff
@@ -78,9 +81,29 @@
 	min_cold_protection_temperature = SPACE_SUIT_MIN_COLD_PROTECTION_TEMPERATURE
 	siemens_coefficient = 0.9
 	species_restricted = list("exclude")
-
+	var/equip_delay = 10 SECONDS
 	var/list/supporting_limbs //If not-null, automatically splints breaks. Checked when removing the suit.
 
+//Delayed equipping of suits
+/obj/item/clothing/suit/space/pre_equip(var/mob/user, var/slot)
```

Comment:
This should go into /obj/item/clothing/ directly so it can be used with other clothing items too.